cirq-google/cirq_google/devices/coupler.py

```diff
@@ -39,52 +39,56 @@ def __init__(self, qubit1: ops.Qid, qubit2: ops.Qid) -> None:
         have an ordering that allows for comparison.
 
         Args:
-            qubit1: The first qubit/Qid of the pair
-            qubit2: The second qubit/Qid of the pair
+            qubit0: The first qubit/Qid of the pair
+            qubit1: The second qubit/Qid of the pair
         """
-        if qubit1 < qubit2:
+        if qubit0 < qubit1:
+            self._qubit0 = qubit0
             self._qubit1 = qubit1
-            self._qubit2 = qubit2
         else:
-            self._qubit1 = qubit2
-            self._qubit2 = qubit1
+            self._qubit0 = qubit1
+            self._qubit1 = qubit0
 
     def __hash__(self) -> int:
```

Comment:
Consider also defining `__eq__`. The base class version using the comparison key is quite slow, so we've specialized `__eq__` in most concrete classes like [`GridQubit`](https://github.com/quantumlib/Cirq/blob/6e19d0da40df4dbcadc8c7ed37a957ff6542f3fa/cirq-core/cirq/devices/grid_qubit.py#L44-L52).

Reply:
Added.